cinema/views.py

```diff
@@ -56,3 +59,30 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderSetPagination(PageNumberPagination):
+    page_size = 1
+    page_size_query_param = "page_size"
+    max_page_size = 3
```

Comment:
it would be better to create separated file for paginator. Do not mix ViewSets & Pagination in one class